src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
there is already a persistence.xml at /webcontent/.... which one is the one working? this one I like it more as the connection looks like it's already done by the application server and this is just referencing the jdbc in payara.

If that is the case:
- postgres dependency in the pom is not needed
- comments should be removed
- the other persistence should be removed